packages/teleport/src/Support/Support.tsx

```diff
@@ -15,14 +15,14 @@
  */
 
 import React from 'react';
-import * as Icons from 'design/Icon';
+import styled from 'styled-components';
 import { Card, Box, Text, Flex } from 'design';
+import * as Icons from 'design/Icon';
 import { FeatureBox } from 'teleport/components/Layout';
 import useTeleport from 'teleport/useTeleport';
 import cfg from 'teleport/config';
-import styled from 'styled-components';
 
-export default function Container() {
+export default function Container({ children }: { children?: JSX.Element }) {
```

Comment:
Can you use `React.ReactNode` here? It's what the official typings use for any React renderable element - https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L799

Reply:
Thanks